typings/single-spa.test-d.ts

```diff
@@ -17,11 +17,6 @@ mountRootParcel(appOrParcel, {
   domElement: document.createElement("div"),
 });
 
-mountRootParcel(appOrParcel, (name: string, location: Location) => ({
```

Comment:
Why was this test deleted?

Could you add new tests to verify the behavior that was broken before this pull request?

Reply:
As mountRootParcel never support function parameter as custom prop, the type checking did not make sense before.

And new test added.

Reply:
Ah I see - it is only applications, not parcels, that allow for custom props as functions. That is something we could perhaps change in the future!